app/src/helpers/PushNotificationsHelper.ts

```diff
@@ -27,34 +33,51 @@ const _foregroundHandler = (): (() => void) => {
  * Permissions Section
  */
 
-const _requestNotificationPermission = async (agent: Agent): Promise<PermissionStatus> => {
-  agent.config.logger.info('Requesting push notification permission...')
+const _requestNotificationPermission = async (): Promise<PermissionStatus> => {
   const result = await request(PERMISSIONS.ANDROID.POST_NOTIFICATIONS)
-  agent.config.logger.info(`push notification permission is now [${result}]`)
   return result
 }
 
-const _checkNotificationPermission = async (agent: Agent): Promise<PermissionStatus> => {
-  agent.config.logger.info('Checking push notification permission...')
-  const result = await check(PERMISSIONS.ANDROID.POST_NOTIFICATIONS)
-  agent.config.logger.info(`push notification permission is [${result}]`)
-  return result
+const formatPermissionIos = (permission: FirebaseMessagingTypes.AuthorizationStatus): NotificationPermissionStatus => {
+  switch (permission) {
+    case messaging.AuthorizationStatus.AUTHORIZED:
+      return NotificationPermissionStatus.GRANTED
+    case messaging.AuthorizationStatus.DENIED:
+      return NotificationPermissionStatus.DENIED
+    case messaging.AuthorizationStatus.PROVISIONAL:
+      return NotificationPermissionStatus.GRANTED
+    default:
+      return NotificationPermissionStatus.UNKNOWN
+  }
 }
 
-const _requestPermission = async (agent: Agent): Promise<void> => {
+const formatPermissionAndroid = (permission: PermissionStatus): NotificationPermissionStatus => {
+  switch (permission) {
+    case RESULTS.GRANTED:
+      return NotificationPermissionStatus.GRANTED
+    case RESULTS.DENIED:
+      return NotificationPermissionStatus.DENIED
+    case RESULTS.BLOCKED:
+      return NotificationPermissionStatus.DENIED
+    default:
+      return NotificationPermissionStatus.UNKNOWN
+  }
+}
+
+const _requestPermission = async (): Promise<NotificationPermissionStatus> => {
```

Comment:
I think _ is meant to be "private" but we don't use the pattern elsewhere in the wallet. Are we able to assume because its not exported its meant to be internal only?

Reply:
ya I didn't add the `_` but I can remove them. I'm assuming that's why they're there